### PR TITLE
Plt#58 rename installation

### LINKED ISSUE #58: Rename Installation model to Country
This had been causing some confusion for us. `Installation`s are referred as `Post`s in all of the views. However, `Posts` can be misleading since it can have meanings like 'posts' in as blog 'posts'. So to clear this out a little bit, I'm thinking of maybe changing its naming into something like 'Nation' or 'Country', since according to #35, 'Posts' have meanings that actually refer to 'Countries'.

As for the naming in the DB table, `Installation` model's `installation` column should be renamed as `name` to keep consistency as well.

Seed files should be filled up with proper examples as well, as we now have an answer on #35 .

### PR DESCRIPTION
**Pull request by wonook**
Issue #58  - Rename Installation model to Country

1.Renamed Installation model to Country
2.Replaced the 'installation' column to 'name'
3.Replaces parts of the views that displayed 'Posts' to 'Countries'
4.Removed the 'contact', 'email', 'address' columns and replaced it with a 'manager'.

Author : @wonook - Won Wook SONG (wsong0512@gmail.com)

This pull request closes #58

### CODE REVIEW

**Comment by abelards** [comment]
All pretty bold changes, I did not see them discussed anywhere?
Anyway if it works and reflects reality better it's a :+1: for me!

**Comment by wonook** [comment]
We've had a discussion on #58 ! This had been the most confusing part of the application when I first came upon this application. It was referred as 'installations' as well as 'posts', so I decided to merge these names into 'country', which was clarified in #35.

**Comment by abelards** [comment]
Indeed! I just hadn't put the pieces together in my head, thanks for the
reminder.
Am 11.08.2015 10:10 schrieb "Won Wook SONG" <notifications@github.com>:

File: app/views/users/_form.html.erb [hunk 1]
```diff
@@ -44,7 +44,7 @@
     <div class="form-group">
       <%= f.label :location, class: "control-label col-sm-3" %>
       <div class="col-sm-3">
-        <%= f.collection_select(:location, Installation.all, :installation, :installation, :include_blank => "None") %>
+        <%= f.collection_select(:location, Country.all, :country, :country, :include_blank => "None") %>
```
Comment (wonook) [inline app/views/users/_form.html.erb]: one of them should be :name

**Pull request merged**